libs/agentc_core/agentc_core/analytics/ddls/all_sessions.sqlpp

```diff
@@ -1,73 +1,70 @@
-CREATE OR REPLACE [ANALYTICS?] FUNCTION
-    `[BUCKET_NAME]`.`[SCOPE_NAME]`.Sessions() {
-    (
-        WITH
-            DeDuplicatedLogs AS (
-                FROM
-                    `[BUCKET_NAME]`.`[SCOPE_NAME]`.`[LOG_COLLECTION_NAME]` AS rsli
-                LET
-                    _id =
-                        -- Note: this is LangChain specific! We need to factor this out later.
-                        CASE
-                            WHEN rsli.kwargs IS NOT MISSING
-                            THEN rsli.kwargs.id
-                            ELSE rsli.content
-                        END
-                SELECT
-                    rsli.*,
-                    ROW_NUMBER()
-                        OVER (
-                            PARTITION BY
-                                _id
-                            ORDER BY
-                                rsli.timestamp ASC
-                        ) AS rn
-            )
+CREATE OR REPLACE [ANALYTICS?] VIEW Sessions AS
```

Comment:
Let's remove the [ANALYTICS?] bits from the queries and replace these with just CREATE OR REPLACE ANALYTICS VIEW (as we'll ultimately have a separate file for the query service UDFs). This means there needs to be updates to how these view DDLs are rendered.